examples/arithmetics/src/language-server/arithmetics-validator.ts

```diff
@@ -9,15 +9,50 @@ export class ArithmeticsValidationRegistry extends ValidationRegistry {
         super(services);
         const validator = services.validation.ArithmeticsValidator;
         const checks: LangiumGrammarChecks = {
-            Division: validator.checkDivByZero
+            Division: validator.checkDivByZero,
+            Definition: validator.checkNormalisable
         };
         this.register(checks, validator);
     }
 }
 export class ArithmeticsValidator {
     checkDivByZero(div: Division, accept: ValidationAcceptor): void {
-        if (isNumberLiteral(div.right) && Number((div.right as NumberLiteral).value) === 0) {
+        if (isNumberLiteral(div.right) && +div.right.value === 0) {
```

Comment:
Is there a specified reason why you start the `div.right.value` statement with a `+`?

Reply:
Type of `div.right.value` is `string`. If I want to compare it with `0`, I should covert it to `number`.

Reply:
I see. Could you add `returns string` to the `NUMBER` terminal rule? It will automatically convert it to a JS `number`.